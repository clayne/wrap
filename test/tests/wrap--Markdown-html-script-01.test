wrap | /dev/null | -u | md-html-script-01.md | 0
