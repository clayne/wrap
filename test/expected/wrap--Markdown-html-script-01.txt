This is some text.  This is some more text.
<script type="text/javascript">
  // Javascript

  document.getElementById("demo").innerHTML = "hello, world";
</script>
This is some text.  This is some more text.
